mobile/Screen/EventDetailScreen.js

```diff
@@ -99,32 +291,117 @@ function EventDetailScreen({
     }
   };
 
+  // TO DO: add 'first' tuesdays, etc.
+  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const dayOfWeek = daysOfWeek[day];
+  const weeksOfMonth = ['First', 'Second', 'Third', 'Fourth'];
+  const weekOfMonth = weeksOfMonth[week - 1];
+
+  const saveIcon = isEventSaved ? require('../assets/detailScreen/saved.png') : require('../assets/detailScreen/unsaved.png');
+
   return (
-    <ScrollView style={styles.container}>
-      <View style={styles.heading}>
-        <Text style={styles.h1}>{title}</Text>
-        <Switch value={isEventSaved} onValueChange={onPressToggleSavedEvent} title="Event Save Button" />
-      </View>
-      <Text onPress={onPressEvent}>{organizations}</Text>
-      <View style={styles.border} />
-      <View style={styles.square} />
-      <Text>{day}</Text>
-      <View style={styles.square} />
-      <Text>{time}</Text>
-      <View style={styles.square} />
-      <Text>{location}</Text>
-      <View style={styles.border} />
-      <Text style={styles.h2}>Event Description</Text>
-      <Text>
-        {' '}
-        {summary}
-        {' '}
-      </Text>
-      <Text style={styles.h2}>Pictures</Text>
+    <ScrollView contentContainerStyle={styles.container}>
```

Comment:
Copying a comment I made below
> Part of the hi-fis includes the header ("Events") being formatted with the font and everything else.  There's an option to hide the header on Screens called `header shown`, which you can set to `false` in the Stack Navigator in `App.js` (which, when you merge main back into this branch, you'll see is now in a component called `AppNavigator.js`) (as in https://github.com/lablueprint/skid-row-arts-alliance/blob/art-details-screen/mobile/Components/AppNavigator.js#L187); this will delete the entire header and allow you to create a custom one like in the hi-fi/final design